Makefile

```diff
@@ -282,7 +282,7 @@ run-docker: build-image
 	# For some tests it is a good idea to add --memory-swappiness=0 --memory-swap $(DOCKER_MEMORY)
 	# This prevents container from swapping but on the other hand, you won't be able
 	# to allocate additional memory needed for the heap dump!
-	docker run --privileged=true -m $(DOCKER_MEMORY) -e USE_PERFORMANCE_LOGGING=true -e APPLICATION_USER=test -e APPLICATION_USER_PASSWORD=test -e KEYSTORE_FILE=/tmp/keystores/keystore_server.jks -e KEYSTORE_PASSWORD=secret -v $(shell pwd)/capacity-tests/src/test/resources:/tmp/keystores -v $(shell pwd)/capacity-tests/target/heapdumps:/tmp/heapdumps $(ADDITIONAL_ARGUMENTS) $(DEV_IMAGE_FULL_NAME)
+	docker run --privileged=true -m $(DOCKER_MEMORY) -e APPLICATION_USER=test -e APPLICATION_USER_PASSWORD=test -e KEYSTORE_FILE=/tmp/keystores/keystore_server.jks -e DEBUG=true -e KEYSTORE_PASSWORD=secret -v $(shell pwd)/capacity-tests/src/test/resources:/tmp/keystores -v $(shell pwd)/capacity-tests/target/heapdumps:/tmp/heapdumps $(ADDITIONAL_ARGUMENTS) $(DEV_IMAGE_FULL_NAME)
```

Comment:
We should probably set `DEBUG=TRUE` for the CI as well

Reply:
Putting it into CI is actually a bit tricky... I would need to put it also in the templates. But maybe that's not a bad idea after all..

Reply:
Good point about the templates. I would say that we definitely shouldn't expose this to the user though. So maybe as part of the Make file we could copy the the templates and add/enable the param before building the image for CI?